src/Package/Impl/DataInspect/DataImport/ImportDataWindow.xaml.cs

```diff
@@ -287,7 +270,7 @@ class DataFramePreviewRowItem {
             await TaskUtilities.SwitchToBackgroundThread();
 
             Encoding encoding = Encoding.GetEncoding(codePage);
-            _tempFilePath = Path.GetTempFileName();
+            _utf8FilePath = Path.Combine(Path.GetTempPath(), Path.GetFileName(file) + ".utf8");
```

Comment:
We'll reuse it then - this prevents us from producing many large oddly named files each time user imports. If they import same file again, we'll reuse the name. Also, readable name looks nicer in REPL. We also can't delete file on close since REPL is still executing (I tried 10 sec timeout, didn't work, it depends on the file size and the CPU speed)